miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -687,7 +688,13 @@ protected LibraryDto createDtoFromParent(Sample item) {
       templatesByProjectId = results.stream()
           .map(sam -> sam.getProject().getId())
           .distinct()
-          .map(projectId -> templateService.listLibraryTemplatesForProject(projectId))
+          .map(projectId -> {
+            try {
+              return libraryTemplateService.listLibraryTemplatesForProject(projectId);
+            } catch (IOException e) {
+              throw new RuntimeException(e);
```

Comment:
Use `WhineyFunction#rethrow` instead (does the same thing):

`.map(WhineyFunction.rethrow(projectId -> libraryTemplateService.listLibraryTemplatesForProject(projectId)))`